test/AspNetCoreModule.Test/Framework/InitializeTestMachine.cs

```diff
@@ -323,30 +446,40 @@ private void PreparePrivateANCMFiles()
                     updateSuccess = false;
                     try
                     {
-                        TestUtility.ResetHelper(ResetHelperMode.KillWorkerProcess);
-                        TestUtility.ResetHelper(ResetHelperMode.StopW3svcStartW3svc);
-                        Thread.Sleep(1000);
+                        //
+                        // NOTE: ANCM schema file can't be overwritten here, if there is any schema change, that should be updated with installing setup
+                        //
 
-                        string from = Path.Combine(outputPath, "x64", "aspnetcore.dll");
-                        TestUtility.FileCopy(from, FullIisAspnetcore_path, overWrite:true, ignoreExceptionWhileDeletingExistingFile:false);
-                        TestUtility.FileCopy(from, IisExpressAspnetcore_path, overWrite: true, ignoreExceptionWhileDeletingExistingFile: false);
+                        if (isIISReady)
```

Comment:
I'm confused by this check. If IIS is ready, we wait?

Reply:
This is not for waiting. Just double-check if the private file is successfully copied and this is only for Full IIS Server. Code is now updated to make it more clear.